pkg/users/users.scss

```diff
@@ -109,3 +109,7 @@
 #groups-create, #account-create {
   min-width: 11rem;
 }
+
```

Comment:
Yeah, we want to make as many fixes as we can in our overrides file and report them upstream, instead of one-offs inside the pages.

Additionally, we should use logical properties instead of left/right for the fixes, so they would ideally work in both RTL and LTR without having to handle RTL. (We may want to convert some of our existing workarounds too.)

https://css-tricks.com/css-logical-properties-and-values/
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties/Margins_borders_padding

Additionally, we probably should also convert our custom CSS to logical properties too, so it will do the right thing without any workarounds for that too.

The gist of it is that you have "inline" for the sides (with start/end replacing left/right or right/left) and "block" for top and bottom (again with start/end replacing top/bottom).

And this works for padding, margin, border, width, height, etc. The biggest gains for RTL would likely be switching our padding and margin from left/right to inline-start/inline-end.

Reply:
Thanks a lot! This is exactly what I was looking for.